pkg/operator/controller/controller_test.go

```diff
@@ -337,11 +337,12 @@ var _ = Describe("Controller", func() {
 				Expect(err).ToNot(HaveOccurred())
 				rule = obj.(*promv1.PrometheusRule)
 				cdiDownAlert := promv1.Rule{
-					Alert: "CdiOperatorDown",
-					Expr:  intstr.FromString("cdi_num_up_operators == 0"),
+					Alert: "CDIOperatorDown",
+					Expr:  intstr.FromString("kubevirt_cdi_num_up_operators == 0"),
```

Comment:
Is there a naming scheme for Kubernetes  metrics? Prometheus suggests to have a type of metrics at the end (_total, _seconds_total, _bytes)
this name could be one of? `kubevirt_cdi_up_operators_number`, `kubevirt_cdi_up_operators_count`, `kubevirt_cdi_up_operators`

Reply:
I have found example here: 
https://github.com/kubernetes/kubernetes/blob/master/test/instrumentation/testdata/stable-metrics-list.yaml

Reply:
Yes makes sense! thx

Reply:
Done